packages/code-studio/src/dashboard/panels/IrisGridPanel.jsx

```diff
@@ -748,7 +746,7 @@ export class IrisGridPanel extends PureComponent {
         advancedSettings,
         pendingDataMap
       ),
-      this.getDehydratedGridState(table, movedColumns, movedRows)
+      this.getDehydratedGridState(model, movedColumns, movedRows)
```

Comment:
You changed it here, but the method signature of `getDehdyratedGridState` still says `table`, update it there as well:
```  
getDehydratedGridState = memoize((table, movedColumns, movedRows) =>
    IrisGridUtils.dehydrateGridState(table, { movedColumns, movedRows })
  );
```